src/components/QuickGrid/DataSelectors.ts

```diff
@@ -79,22 +79,24 @@ const filterRows = (rows: Array<any>, columns: Array<GridColumn>, searchText: st
     if (!searchText) {
         return rows;
     }
-
     let filterText = searchText.toLowerCase();
-
     let members = columns.map(col => col.valueMember);
-
     const newRows = rows.filter(row => {
         let visible = false;
         for (let value of members) {
-            if (row[value].toString().toLowerCase().indexOf(filterText) !== -1) {
-                visible = true;
-                break;
+            let objectProps = value.split('.');
```

Comment:
how's the performance impact of these changes?

Reply:
Switched from 100k to 200k and no impact.